src/cmlibs/utils/__init__.py

```diff
@@ -1,2 +1,2 @@
 
-__version__ = '0.6.0'
+__version__ = '0.6.1'
```

Comment:
We change the version just prior to making a release, it isn't helpful to change the version number now.